examples/src/counter.js

```diff
@@ -1,11 +1,10 @@
-import { NearContract, NearBindgen, near, call, view } from 'near-sdk-js'
+import { NearBindgen, near, call, view, initialize } from 'near-sdk-js'
 import { isUndefined } from 'lodash-es'
 
-@NearBindgen
-class Counter extends NearContract {
-    constructor({ initial = 0 }) {
-        super()
-        this.count = initial
+@NearBindgen({})
```

Comment:
Can this be written as `@NearBindgen()` or simply `@NearBindgen`?

Reply:
Now `@NearBidgen` is defined as 
```JS
export function NearBindgen({ requireInit = false }: { requireInit?: boolean }) {
```
we are using named optional parameters.
It allows us to omit `requireInit` or other parameters and use the default value. Since it is a 'named' parameter, we can omit some of them. This design is extendable and this is why it needs to use a bit more complicated syntaxis.
 ```JS
@NearBindgen({})
```
Alternative 1: make all parameters required, NearBingen will always look like this:
```JS
@NearBindgen({ requireInit: true/false })
```

Alternative 2:  we can drop named parameters, but the design will not be intuitive:
```JS
@NearBindgen(true)
```

Reply:
Got it! I think your implementation is better than the alternatives

Reply:
`@NearBindgen`, `@NearBindgen()` and `@NearBindgen({ requireInit: true })`

all work with this patch:

```patch
diff --git a/src/build-tools/near-bindgen-exporter.js b/src/build-tools/near-bindgen-exporter.js
index 8cd2780..6a34c9c 100644
--- a/src/build-tools/near-bindgen-exporter.js
+++ b/src/build-tools/near-bindgen-exporter.js
@@ -5,7 +5,7 @@ export default function () {
     visitor: {
       ClassDeclaration(path) {
         let classNode = path.node;
-        if (classNode.decorators && classNode.decorators[0].expression.callee.name == 'NearBindgen') {
+        if (classNode.decorators && (classNode.decorators[0].expression.name || classNode.decorators[0].expression.callee.name) == 'NearBindgen') {
           let classId = classNode.id;
           let contractMethods = {};
           for (let child of classNode.body.body) {
diff --git a/src/near-bindgen.ts b/src/near-bindgen.ts
index 9cf0d0e..73a175e 100644
--- a/src/near-bindgen.ts
+++ b/src/near-bindgen.ts
@@ -9,7 +9,9 @@ export function call(target: Object, key: string | symbol, descriptor: TypedProp
 export function view(target: Object, key: string | symbol, descriptor: TypedPropertyDescriptor<Function>): void {
 }

-export function NearBindgen({ requireInit = false }: { requireInit?: boolean }) {
+const DEFAULT_ARGS = { requireInit: false };
+
+export function NearBindgen({ requireInit = false }: { requireInit?: boolean } = DEFAULT_ARGS) {
     return <T extends { new(...args: any[]): {} }>(target: T) => {
         return class extends target {
             static _create() {
```